less/_grid.less

```diff
@@ -114,18 +114,50 @@
 		@arrayVal: extract(@list, @index);
 		// this declaration block gets output. once per key in the @list array 
 		@pcnt: percentage(@arrayVal / @listLength);
-		// this declaration block gets output. once per key in the @list array 
+		// this declaration block gets output. once per key in the @list array
 		.col@{arrayVal} {
 			display: block;
 			float: left;
 			padding: 0 @grid-sides;
-			@media @mq-sm, @mq-md {
-				width: 100%;
+			width: 100%;
+			// THREE COLUMN
+			& when (@arrayVal = 4) {
```

Comment:
Works with three column layouts. It sets the first two columns to `50%` and the last column to `100%`